### PR TITLE
working

### PR DESCRIPTION
**Pull request by 4var1**

### CODE REVIEW

**Comment by 4var1** [comment]
I have no idea why the previous PRs were causing conflicts with pins.c - so I created a fresh branch and copied the changes in. All tested ok - and now no merge conflicts!

File: EBike_wireless_TSDZ2/firmware/.vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -3,21 +3,25 @@
         {
             "name": "nRF SDK 16.0.0",
             "includePath": [
-                "${workspaceFolder}/**"
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/include/**",
+                "C:/Program Files (x86)/GNU Tools ARM Embedded/7 2018-q2-update/arm-none-eabi/include/**",
```
Comment (casainho) [inline EBike_wireless_TSDZ2/firmware/.vscode/c_cpp_properties.json:8]: remove this as it is specific to you

File: EBike_wireless_TSDZ2/firmware/Makefile [hunk 2]
```diff
@@ -17,7 +17,7 @@ BOOTLOADER_DIR = $(PROJ_DIR)/../../EBike_wireless_bootloader/firmware
 BOOTLOADER_HEX = $(BOOTLOADER_DIR)/_build/TSDZ2_bootloader_with_sd.hex
 
 APP_VER_MAJOR = 0
-APP_VER_MINOR = 6
+APP_VER_MINOR = 5
```
Comment (casainho) [inline EBike_wireless_TSDZ2/firmware/Makefile:20]: keep the previous 0.6 version

File: EBike_wireless_TSDZ2/firmware/Makefile [hunk 3]
```diff
@@ -203,12 +208,14 @@ INC_FOLDERS += \
 	$(SDK_ROOT)/components/ble/ble_advertising \
   $(SDK_ROOT)/components/ant/ant_state_indicator \
   $(SDK_ROOT)/components/ant/ant_key_manager \
+  $(SDK_ROOT)/components/ant/ant_key_manager/config \
```
Comment (casainho) [inline EBike_wireless_TSDZ2/firmware/Makefile:211]: remove this, not needed

Comment (4var1) [reply EBike_wireless_TSDZ2/firmware/Makefile:211]: OK - i'll close and resubmit - I'll also merge my changes better as I was about to accidentally back out some changes from rananna i think....

**Comment by 4var1** [comment]
Oh - something unexpected has happened - there are changes in main.c not made by me th

**Pull request closed without merging**